### PR TITLE
Added api version in authn and authz example.

### PR DESCRIPTION
**Pull request by apollocse**
Updated authn and authz example. For novice user using these examples are very helpful. Without api version api-server fails to restart.

### CODE REVIEW

**Comment by neolit123** [comment]
seems good.
/sig auth

File: content/en/docs/reference/access-authn-authz/webhook.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ file format. Within the file "users" refers to the API Server webhook and
 A configuration example which uses HTTPS client auth:
 
 ```yaml
+#Kubernetes API version
+apiVersion: v1
```
Comment (tengqm) [inline content/en/docs/reference/access-authn-authz/webhook.md]: Can you help point me to the issue/pr/source/... where the 'apiVersion' field becomes required? The YAML is a kubeconfig struct where apiVersion is not strictly required IIRC.

**Comment by mistyhacks** [comment]
/hold

(until above question is answered)

File: content/en/docs/reference/access-authn-authz/webhook.md (hunk 1 above)
Comment (apollocse) [reply content/en/docs/reference/access-authn-authz/webhook.md]: I have seen an intermittent issues restarting api-server (fails to restart) after configuring web-hook. But it work flawlessly when api version is mentioned.
Not sure if there is an existing issue for this.

Comment (tengqm) [reply content/en/docs/reference/access-authn-authz/webhook.md]: ping @deads2k, @lavalamp, PTAL. Thanks.

Comment (lavalamp) [reply content/en/docs/reference/access-authn-authz/webhook.md]: You should always specify it IMO. We make assumptions given the context it's used in but better to specify it.

apiVersion includes the group, if we want to be technical.

You should also specify the kind.

These lines don't merit a comment and if you keep the comment anyway, place a space after the #.

Comment (apollocse) [reply content/en/docs/reference/access-authn-authz/webhook.md]: @lavalamp I fixed comment. Although I had no luck with grop and kind. For Authentication I tried
apiVersion: authentication.k8s.io/v1
kind: TokenReview

But with this api-server fails to restart. Am I missing anything here?

**Comment by Bradamant3** [comment]
@lavalamp it looks as though we still have an outstanding question about exactly how to call out version/group/kind. Can you help out further? Thank you!

**Comment by zacharysarah** [comment]
@neolit123 👋 Are you able to weigh in on the [discussion above](https://github.com/kubernetes/website/pull/9696#discussion_r206736541)? ☝️ Thanks in advance!

File: content/en/docs/reference/access-authn-authz/webhook.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ file format. Within the file "users" refers to the API Server webhook and
 A configuration example which uses HTTPS client auth:
 
 ```yaml
+# Kubernetes API version
+apiVersion: v1
```
Comment (neolit123) [inline content/en/docs/reference/access-authn-authz/webhook.md]: [1]

make this into:
```
# Kubernetes API version
apiVersion: v1
# kind of the API object
kind: Config
```

^ adds `kind` as well.

File: content/en/docs/reference/access-authn-authz/authentication.md [hunk 3]
```diff
@@ -408,6 +408,8 @@ file format. Within the file, `clusters` refers to the remote service and
 `users` refers to the API server webhook. An example would be:
 
 ```yaml
+# Kubernetes API version
+apiVersion: v1
```
Comment (neolit123) [inline content/en/docs/reference/access-authn-authz/authentication.md]: [1]

**Comment by zacharysarah** [comment]
Thanks @neolit123! 🙇 I added your feedback directly.

/lgtm
/approve

**Pull request merged**